assets/xml/objects/object_um.xml

```diff
@@ -85,7 +94,19 @@
         <Texture Name="object_um_Tex_014CD0" OutName="tex_014CD0" Format="ci8" Width="32" Height="64" Offset="0x14CD0" />
         <Texture Name="object_um_Tex_0154D0" OutName="tex_0154D0" Format="ci8" Width="32" Height="64" Offset="0x154D0" />
         <Texture Name="object_um_Tex_015CD0" OutName="tex_015CD0" Format="rgba16" Width="32" Height="32" Offset="0x15CD0" />
-        <!-- <Blob Name="object_um_Blob_0164D0" Size="0x2000" Offset="0x164D0" /> -->
+
+        <Texture Name="gUmCremiaEyeOpenTex" OutName="cremia_eye_open" Format="ci8" Width="32" Height="32" Offset="0x164D0" TlutOffset="0x130D0" />
+        <Texture Name="gUmCremiaEyeHalfTex" OutName="cremia_eye_half" Format="ci8" Width="32" Height="32" Offset="0x168D0" TlutOffset="0x130D0" />
+        <Texture Name="gUmCremiaEyeClosedTex" OutName="cremia_eye_closed" Format="ci8" Width="32" Height="32" Offset="0x16CD0" TlutOffset="0x130D0" />
+        <Texture Name="gUmCremiaEyePleasedTex" OutName="cremia_eye_pleased" Format="ci8" Width="32" Height="32" Offset="0x170D0" TlutOffset="0x130D0" />
+        <Texture Name="gUmCremiaEyeAngryTex" OutName="cremia_eye_angry" Format="ci8" Width="32" Height="32" Offset="0x174D0" TlutOffset="0x130D0" />
+        <Texture Name="gUmCremiaEyeSadTex" OutName="cremia_eye_sad" Format="ci8" Width="32" Height="32" Offset="0x178D0" TlutOffset="0x130D0" />
+
+        <Texture Name="gUmMouthNeutralTex" OutName="mouth_neutral" Format="ci8" Width="32" Height="16" Offset="0x17CD0" TlutOffset="0x130D0" />
+        <Texture Name="gUmMouthHappyTex" OutName="mouth_happy" Format="ci8" Width="32" Height="16" Offset="0x17ED0" TlutOffset="0x130D0" />
+        <Texture Name="gUmMouthSadTex" OutName="mouth_sad" Format="ci8" Width="32" Height="16" Offset="0x180D0" TlutOffset="0x130D0" />
+        <Texture Name="gUmMouthOpenTex" OutName="mouth_open" Format="ci8" Width="32" Height="16" Offset="0x182D0" TlutOffset="0x130D0" />
```

Comment:
And similarly, I called them this:
```xml
        <Texture Name="gCremiaMouthNormalTex" OutName="cremia_mouth_normal" Format="rgba16" Width="32" Height="16" Offset="0x14AD8" />
        <Texture Name="gCremiaMouthSlightSmileTex" OutName="cremia_mouth_slight_smile" Format="rgba16" Width="32" Height="16" Offset="0x14ED8" />
        <Texture Name="gCremiaMouthFrownTex" OutName="cremia_mouth_frown" Format="rgba16" Width="32" Height="16" Offset="0x152D8" />
        <Texture Name="gCremiaMouthHangingOpenTex" OutName="cremia_mouth_hanging_open" Format="rgba16" Width="32" Height="16" Offset="0x156D8" />
```
Again, which do you prefer?

Reply:
I think I prefer "Happy" over "SlightSmile", everything else seems better